.gitignore

```diff
@@ -13,3 +13,4 @@ src/ADAL.PCL.Android/ADAL.PCL.Android.csproj.bak
 AppPackages
 Microsoft.IdentityModel.Clients.ActiveDirectory.XML
 *.htm
+.nugetPackageRoot/
```

Comment:
By default, the targets will create the NuGet folder layout under this folder (although you can specify an alternative location by setting the appropriate build property).